### PR TITLE
Remove useless codes 

### PR DESCRIPTION
**Pull request by 2997ms**
Fix the useless code in RadioComponentController.cc

### CODE REVIEW

**Comment by 2997ms** [comment]
@DonLakeFlyer

File: src/AutoPilotPlugins/Common/RadioComponentController.cc [hunk 1]
```diff
@@ -867,9 +867,7 @@ void RadioComponentController::_loadSettings(void)
     _transmitterMode = settings.value(_settingsKeyTransmitterMode, 2).toInt();
     settings.endGroup();
 
-    if (_transmitterMode != 1 || _transmitterMode != 2) {
-        _transmitterMode = 2;
-    }
+    _transmitterMode = 2;
```
Comment (DonLakeFlyer) [inline src/AutoPilotPlugins/Common/RadioComponentController.cc:870]: Not sure why you think this is not needed. The current code is correct and required to validate the two valid settings from _transmitterMode.

Comment (2997ms) [reply src/AutoPilotPlugins/Common/RadioComponentController.cc:870]: `if (_transmitterMode != 1 || _transmitterMode != 2)`
This is always true.

Comment (DonLakeFlyer) [reply src/AutoPilotPlugins/Common/RadioComponentController.cc:870]: Ah, right it's screwed up. Should be `if (!(_transmitterMode == 1 || _transmitterMode == 2))`

Comment (2997ms) [reply src/AutoPilotPlugins/Common/RadioComponentController.cc]: @DonLakeFlyer Addressed

Comment (2997ms) [reply src/AutoPilotPlugins/Common/RadioComponentController.cc]: @DonLakeFlyer

**Comment by DonLakeFlyer** [comment]
Great thanks

**Pull request merged**